tests/models/registry.py

```diff
@@ -215,10 +224,11 @@ def check_available_online(
     "OrionForCausalLM": _HfExamplesInfo("OrionStarAI/Orion-14B-Chat",
                                         trust_remote_code=True),
     "PersimmonForCausalLM": _HfExamplesInfo("adept/persimmon-8b-chat"),
-    "PhiForCausalLM": _HfExamplesInfo("microsoft/phi-2"),
+    "PhiForCausalLM": _HfExamplesInfo("microsoft/phi-2", v0_only=True),
```

Comment:
QQ - do we know why these are failing? Is it the head_dim?

Reply:
Yes it is head_dim for most of these. The only exceptions IIRC are Kimi and Pixtral, which have tracking issue